pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -175,6 +177,11 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 
+		_, err = r.context.Clientset.CoreV1().Secrets(namespace).Get(context.TODO(), crashCollectorKeyName, metav1.GetOptions{})
+		if err != nil {
+			return reconcile.Result{}, errors.Wrapf(err, "could not list the %q secrets", crashCollectorKeyName)
```

Comment:
this line is making logs(INFO) verbose as it will keep printing this until secret is found.


a small snippet...this keeps printing until secrete is create so basically until mons are in quorum
```
2021-05-10 08:35:04.278359 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:04.678443 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:04.882227 I | op-mon: mons running: [a b]
2021-05-10 08:35:05.078560 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:05.115361 I | op-mon: Monitors in quorum: [a]
2021-05-10 08:35:05.115385 I | op-mon: mons created: 2
2021-05-10 08:35:05.278613 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:05.323176 I | op-mon: waiting for mon quorum with [a b]
2021-05-10 08:35:05.482050 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:05.880374 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:06.092882 I | op-mon: mons running: [a b]
2021-05-10 08:35:06.278373 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:06.291846 W | op-mon: failed to find initial monitor b in mon map
2021-05-10 08:35:06.291861 W | op-mon: monitor b is not in quorum list
2021-05-10 08:35:06.601784 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:07.245684 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:07.276075 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:08.530435 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:11.094073 E | ceph-crashcollector-controller: could not list the "rook-ceph-crash-collector-keyring" secrets: secrets "rook-ceph-crash-collector-keyring" not found
2021-05-10 08:35:11.300801 I | op-mon: mons running: [a b]
```
any suggestion?